test/unit/player.test.js

```diff
@@ -1385,6 +1385,34 @@ QUnit.test('sets lang attribute on player el', function(assert) {
   }
 });
 
+QUnit.test('language changed should trigger languagechange event', function(assert) {
```

Comment:
I think another test where we change the language multiple times would be good to have. Twice should suffice. Just to make sure that we don't get stuck in the new state.